### ISSUE TITLE
Improving task868_cfq_mcd1_explanation_to_sql.json

### ISSUE THREAD

**Issue by danyaljj** [allenai/natural-instructions#492]
This task seems to be quite difficult. Wondering if there is a way for us to improve it.

**Comment by swarooprm** [comment]
@ujjwalaananth, can you have a look at this?

**Comment by ujjwalaananth** [comment]
Difficult in terms of the explanation not being clear, or difficult as in the expected answers are too complicated? In the latter case, maybe removing { } and escape sequences like \n would make it much clearer, as they are only there for human readability I believe and the resulting query would still be correct, but more straightforward to learn.

**Comment by ujjwalaananth** [comment]
For example, for the input "Did M0 's art director , director , producer , and writer executive produce , write , produce , and direct M1"

current output is 
"SELECT count(*) WHERE {\n?x0 ns:film.director.film M0 .\n?x0 ns:film.director.film M1 .\n?x0 ns:film.film_art_director.films_art_directed M0 .\n?x0 ns:film.producer.films_executive_produced M1 .\n?x0 ns:film.producer.film|ns:film.production_company.films M0 .\n?x0 ns:film.producer.film|ns:film.production_company.films M1 .\n?x0 ns:film.writer.film M0 .\n?x0 ns:film.writer.film M1\n}"

but maybe this could be simplified to   
"SELECT count(*) WHERE x0 ns:film.director.film M0 . x0 ns:film.director.film M1 . x0 ns:film.film_art_director.films_art_directed M0 . x0 ns:film.producer.films_executive_produced M1 . x0 ns:film.producer.film|ns:film.production_company.films M0 . x0 ns:film.producer.film|ns:film.production_company.films M1 . x0 ns:film.writer.film M0 . x0 ns:film.writer.film M1"

(\n and braces removed)
(however I cannot guarantee this query is correct.)

**Comment by danyaljj** [comment]
Right, I am actually not sure. For now, let's keep the issue open, in case we have good ideas for solving it.
